Formula/wolfmqtt.rb

```diff
@@ -0,0 +1,38 @@
+class Wolfmqtt < Formula
+  desc "Small, fast, portable MQTT client C implementation"
+  homepage "https://github.com/wolfSSL/wolfMQTT"
+  url "https://github.com/wolfSSL/wolfMQTT.git",
+      :tag      => "v1.4",
+      :revision => "af3d2926773b2c97f5e3a86ea2562e339a91b747"
+  head "https://github.com/wolfSSL/wolfMQTT.git"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
+  depends_on "wolfssl"
+
+  def install
+    args = %W[
+      --disable-silent-rules
+      --disable-dependency-tracking
+      --infodir=#{info}
+      --mandir=#{man}
+      --prefix=#{prefix}
+      --sysconfdir=#{etc}
+      --enable-nonblock
+      --enable-mt
+      --enable-mqtt5
+      --enable-propcb
+      --enable-sn
+    ]
+
+    system "./autogen.sh"
+    system "./configure", *args
+    system "make"
+    system "make", "install"
+  end
+
+  test do
+    system bin/"wolfmqtt-config", "--cflags", "--libs", "--prefix"
```

Comment:
Please check some output here and possibly run a test that's more in line with what users are expected to run.